bitnami/mariadb/templates/update-password/new-secret.yaml

```diff
@@ -0,0 +1,28 @@
+{{- /*
+Copyright Broadcom, Inc. All Rights Reserved.
+SPDX-License-Identifier: APACHE-2.0
+*/}}
+
+{{- if and .Values.passwordUpdateJob.enabled (eq ( include "mariadb.createSecret" . ) "true") (not ( include "mariadb.createPreviousSecret" . )) (not .Values.passwordUpdateJob.previousPasswords.existingSecret) }}
```

Comment:
Can we replace `eq ( include "mariadb.createSecret" . ) "true"` with `(include "mariadb.createSecret" . )` or does it fail?

Reply:
As it is a major bump, let me change the createSecret helper because right now it is returning a string with "true" or "false"

Reply:
Actually not, I do not know why we were using `eq`. Changing it